### ISSUE TITLE
Not working on ACS

### ISSUE THREAD

**Issue by seikbosna** [Stefanuk12/Aiming#11]
**Describe the bug**
Everything executes fine but I tried every method individually and none of them work on the ACS. As I said, I tried them individually. I am a little confused by all of this so If you know good configuration for ACS or how to make silent aim work on ACS tell me. I tried in on ACS 1.7.5.

**Comment by Stefanuk12** [comment]
Are you getting a tracer?

**Comment by seikbosna** [comment]
> Are you getting a tracer?

If you mean tracer like that line, like on esp, then yes, but I only got it like 1 time for a second before it disappeared

**Comment by Stefanuk12** [comment]
What game? It's probably because that game is not supported and needs a patch. If you know how to code, you can make one and then create a pull request to add it.

**Comment by seikbosna** [comment]
> What game? It's probably because that game is not supported and needs a patch. If you know how to code, you can make one and then create a pull request to add it.

Alpha's planks. I just started learning lua and I am not good at coding yet. I just know the basics. If you can, please make a patch for that game or the ACS as a whole, because it is not working on any other ACS games that use ACS 1.7.5 (I didn't test it on ACS 2.0)

**Comment by ShoziOffical** [comment]
yep i can confirm this doesnt work on ACS games like Operation Overlord for example, Raycast should work on ACS games for the other silent aim exploits but this doesnt for some reason

**Comment by seikbosna** [comment]
> yep i can confirm this doesnt work on ACS games like Operation Overlord for example, Raycast should work on ACS games for the other silent aim exploits but this doesnt for some reason

Can you send me the silent aim script that works on ACS if you have one?

**Comment by seikbosna** [comment]
@warface1234455 @Stefanuk12 still not sure if people here see the replies, that is why I am pinging, sorry for interupting btw

**Comment by ShoziOffical** [comment]
> @warface1234455 @Stefanuk12 still not sure if people here see the replies, that is why I am pinging, sorry for interupting btw

@seikbosna here `loadstring(game:HttpGet("https://raw.githubusercontent.com/Averiias/Universal-SilentAim/main/main.lua"))()`

**Comment by seikbosna** [comment]
> https://raw.githubusercontent.com/Averiias/Universal-SilentAim/main/main.lua

@warface1234455 thank you very much bro

**Comment by seikbosna** [comment]
> > @warface1234455 @Stefanuk12 still not sure if people here see the replies, that is why I am pinging, sorry for interupting btw
> 
> @seikbosna here `loadstring(game:HttpGet("https://raw.githubusercontent.com/Averiias/Universal-SilentAim/main/main.lua"))()`

@warface1234455 
![image](https://user-images.githubusercontent.com/123584898/218811822-7b62a6d3-ecd3-49f8-be79-2c784478249b.png)
this pops up in the dev console when I try to execute it. It isn't working on Alpha's, do you know how to solve this issue

**Comment by ShoziOffical** [comment]
huh it was working for me 3 days ago, that happens to me too very odd

**Comment by warface1234455** [comment]
but anyways, use this one this works on ACS `https://pastebin.com/raw/GDMU2jap` @seikbosna

**Comment by seikbosna** [comment]
> huh it was working for me 3 days ago, that happens to me too very odd

@warface1234455 oh, idk then, can you try executing it again please?

**Comment by ShoziOffical** [comment]
> you

It just doesn't work anymore for some reason, the script github is archived

**Comment by PeasIn** [comment]
> It just doesn't work anymore for some reason, the script github is archived

i think its because the UI got deleted

**Comment by ShoziOffical** [comment]
> > It just doesn't work anymore for some reason, the script github is archived
> 
> i think its because the UI got deleted

The github script is archived long ago so it's only readable idk how they manage to remove it now

**Comment by PeasIn** [comment]
> The github script is archived long ago so it's only readable idk how they manage to remove it now

i took a look at the line 243 and they seem to use someone's else UI and it got deleted. i think if someone is smart enough to transfer the script to a new UI it will work

**Comment by ShoziOffical** [comment]
Ahh i see, was one of the best silent aim scripts too bad someome decides to delete the ui for some reasom

**Comment by ShoziOffical** [comment]
@Stefanuk12 if you could try to copy the raycast method in the script above, it could fix the acs issues, oh wait you are one of the dev of that script i hope you could fix this new one

**Comment by seikbosna** [comment]
> @Stefanuk12 if you could try to copy the raycast method in the script above, it could fix the acs issues, oh wait you are one of the dev of that script i hope you could fix this new one

@warface1234455 @PeasIn @Stefanuk12 This script is broken too, you cannot select the methods anymore

**Comment by seikbosna** [comment]
@warface1234455 @PeasIn People are again getting caught with silent aim in FPS games, perhaps, there is fixed version

**Comment by seikbosna** [comment]
@warface1234455 @PeasIn Do you know a good HBE script by any chance?

**Comment by PeasIn** [comment]
@seikbosna @warface1234455 I DID IT, I FIXED THE UI (silent aim works (i tested))
`loadstring(game:HttpGet("https://raw.githubusercontent.com/PeasIn/Universal-SilentAim/main/main.lua"))()`

**Comment by seikbosna** [comment]
> @seikbosna @warface1234455 I DID IT, I FIXED THE UI (silent aim works (i tested)) `loadstring(game:HttpGet("https://raw.githubusercontent.com/PeasIn/Universal-SilentAim/main/main.lua"))()`

@PeasIn  omg bro, thank you

**Comment by seikbosna** [comment]
> @seikbosna @warface1234455 I DID IT, I FIXED THE UI (silent aim works (i tested)) `loadstring(game:HttpGet("https://raw.githubusercontent.com/PeasIn/Universal-SilentAim/main/main.lua"))()`

@PeasIn congrats on fixing it but it is still not working on acs, add me on dizzy when you have time btw, I want to ask you something

**Comment by PeasIn** [comment]
> > @seikbosna @warface1234455 I DID IT, I FIXED THE UI (silent aim works (i tested)) `loadstring(game:HttpGet("https://raw.githubusercontent.com/PeasIn/Universal-SilentAim/main/main.lua"))()`
> 
> @PeasIn congrats on fixing it but it is still not working on acs, add me on dizzy when you have time btw, I want to ask you something

PeasIn#1411 is my tag

**Comment by ghost** [comment]
it works wdym


FindPartOnRayWithIgnoreList
Teamcheck disabled


try that

**Comment by MaguirePool** [comment]
or alternatively

https://streamable.com/gr4q2j
